src/utils.rs

```diff
@@ -193,6 +193,9 @@ mod tests {
             fs::remove_file(wallet_path).unwrap();
         }
     }
+
+    /// Create a wallet file with optional wallet content.
```

Comment:
I guess I meant, _what is_ content? I can see that it's optional, but I don't understand what content is made up of.

Reply:
Content could be anything, just populates file with data. Doesn't have to be specific format.
P.s. Feel free to suggest a description (or improvement)

Reply:
How about something like: 

```rust
/// Represents the possible serialized states of a wallet.
/// Used primarily for simulating wallet creation and serialization processes.
enum WalletSerializedState {
    Empty,
    WithData(String),
}

/// Simulates the serialization of a wallet to a file, optionally including dummy data.
/// Primarily used to test if checks for wallet file existence are functioning correctly.
fn serialize_wallet_to_file(wallet_path: &Path, state: WalletSerializedState) {
    // Create the wallet file if it does not exist.
    if !wallet_path.exists() {
        fs::File::create(wallet_path).unwrap();
    }

    // Write content to the wallet file based on the specified state.
    if let WalletSerializedState::WithData(data) = state {
        fs::write(wallet_path, data).unwrap();
    }
}
```

Leaving this as a suggestion only maybe Josh has something else in mind

Reply:
updated: https://github.com/FuelLabs/forc-wallet/pull/220/commits/f8124e8d4e100bcfdfd7a44d6af5d91bf318b68f

Reply:
thanks; updated: https://github.com/FuelLabs/forc-wallet/pull/220

Reply:
Thanks Kaya, that's perfect.